File: problems/p1/main.go

/**
 * Ron Nathaniel
 * July 26 2020
 */
 
package main

import (
  "fmt"
)


func Total(limit int) (total int) {
  total = 0
  for i := 0; i < limit; i++ {
    if i % 3 == 0 || i % 5 == 0 {
      total += i
    }
  }
  return
}

func main() {
  t := Total(1000)
  fmt.Println(t)
}

File: problems/p1/main.js

/**
 * Ron Nathaniel
 * July 26 2020
 */
 
main = (limit = 1000) => {
  let total = 0
  for (let i = 0; i < limit; ++i) {
    if (i % 3 == 0 || i % 5 == 0) {
      total += i;
    }
  }
  return total;
}

const t = main()
console.log(t)
